config/webpack.client.prod.js

```diff
@@ -25,6 +25,7 @@ const config = Object.assign({}, defaults, {
   }),
   plugins: defaults.plugins.concat([
     new ManifestPlugin({ fileName: 'manifest.json' }),
+    new ManifestPlugin({ fileName: `manifest.${Date.now()}.json` }),
```

Comment:
This is be problematic since we use manifest.json for knowing the output filenames:
https://github.com/firstlookmedia/react-scripts/blob/master/lib/manifest.js#L25